
float64 bot_x
float64 bot_y
float64 bot_theta
float64 bot_linear
float64 bot_angular

